Carmunication/Controllers/SessionStart/SessionStartViewController.swift

```diff
@@ -93,6 +95,10 @@ extension SessionStartViewController {
             make.leading.trailing.equalToSuperview().inset(20)
             make.height.lessThanOrEqualTo(467).priority(.low)
         }
+        sessionStartMidNoGroupView.snp.makeConstraints { make in
+            make.leading.top.trailing.bottom.equalToSuperview().inset(20)
```

Comment:
```suggestion
            make.edges.equalToSuperview().inset(20)
```
이렇게 줄여 쓰면 더 짧아질 것 같습니다🙃

Reply:
아앗 수정하겠슴다 ㅋㅋㅋ